docs/source/es/_toctree.yml

```diff
@@ -55,13 +55,17 @@
   - local: custom_models
     title: Compartir modelos personalizados
   - local: run_scripts
+  - local: trainer
+    title: Entrenador
```

Comment:
There are two `title`'s here which is causing the doc-builder to fail. I think we just want to keep `Entrenador` and remove `Entrenamiento con scripts`?

Reply:
Will change it :) Thanks!